test/test.helper.ts

```diff
@@ -0,0 +1,3 @@
+export * as factory from './factories/index'
+
+export { Product } from '../src/models'
```

Comment:
What is the purpose of this barrel file?

Since you're already exporting the `factory` from its index, you can use it wherever needed. The same applies to the `Product` model you can directly use it from the `index.ts` where it's being exported.

Reply:
I want to create a `TestBase` file from which you can export whatever you need so all specs only have one import, making the specs I think, more readable since we can define blocks of imports on the helper to organize from where we are pulling the objects. Did not called it test or base since it would look weird `test.base` and if I used `base.test.ts` it would recognize the file as a `spec` file meaning it would try to run it when the tests are run.